### ISSUE TITLE
Set Material Attribute Hue Cesium Color

### ISSUE THREAD

**Issue by softyoda** [CesiumGS/cesium-unreal#977]
Hi, I'm trying to get the cesium base material and change its hue/bright/contrast to match albedo of my model. But I struggle with set material override.

I've followed those tutorials : https://cesium.com/learn/unreal/unreal-edit-materials/

![parsecd-Parsec-11-47_11-octobre-22_1355-85170_1174-373](https://user-images.githubusercontent.com/13988751/195057847-84c07af5-fe2c-459e-b71d-9c5ef9d06f7e.jpg)

But this gives me black material, it should just pass base color to base color (without any hue or any modification) I don't know where is my issue.

**Comment by softyoda** [comment]
Ok, it's weird.

I think I found a way, i don't know if it is the best one, but it work on my case.

I changed the ML_CesiumRasterOverlay material and changed it like that : 
![image](https://user-images.githubusercontent.com/13988751/195067733-8aec8f05-e392-4954-b801-171a30d21271.png)

Then in the layer parameters, as there is 4 time (Clipping, Overlay2, Overlay1, Overlay0) idk why, but only changing scalar parameters on the last one (Overlay 0) works.
